panels/history/ha-panel-history.html

```diff
@@ -202,8 +202,8 @@
 
   stateHistoryObserver(newVal) {
     setTimeout(() => {
-      if (newVal === this.stateHistoryInput) {
-        // Input didn't change
+      if (newVal !== this.stateHistoryOutput) {
```

Comment:
Revert this equals change. When input changes, we should only write it to output if the input is still the same. If this property is triggered 5 times, the old behavior is that only the last change will win. With this change, all will be set.